### DISCUSSION TITLE
store-product 和 store-home 组件，使用原生写法不生效

### DISCUSSION THREAD

**Discussion by hpfgit** [NervJS/taro#17087] [Q & A]
有什么方法可以支持react  使用 store-product、store-home 组件 生效

**Comment by wyxznl** [comment]
请问问题解决了吗

**Reply by TheKonka** [reply]
taro-plugin-inject

**Comment by hpfgit** [comment]
解决了，版本库的问题，




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "NervJS/taro"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年1月14日(星期二) 晚上11:54
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [NervJS/taro] store-product 和 store-home 组件，使用原生写法不生效 (Discussion #17087)





 
请问问题解决了吗
